modules/serverless-logic-overview-error-handling.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-error-handling_{context}"]
```

Comment:
ftirados@redhat.com would you mind take a look a this error handling section?

Reply:
@ftirados@redhat.com can you please review the error handling section?